### ISSUE TITLE
Overriding a method with positional-only parameters is not flagged as an error

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#3203]
**Describe the bug**
Given the following code with `reportIncompatibleMethodOverride` set to `"error"`:
```python
from __future__ import annotations


class Thing:
    def method(self, one: int, two: str) -> None:
        print(one, two)


class OtherThing(Thing):
    def method(self, one: int, two: str, /) -> None:
        super().method(one, two)


def call_method(thing: Thing) -> None:
    thing.method(one=1, two='two')


call_method(OtherThing())
```

Line 10 does not generate an error, but a runtime error is generated for line 18 since `OtherThing.method()` does not accept named arguments.

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**
An `reportIncompatibleMethodOverride` error should be generated for line 10.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command line

Which version?
1.1.229

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by erictraut** [comment]
This is fixed in pyright 1.1.231, which I just published. It will also be included in the next release of pylance.
